extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,10 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+// DO NOT SUBMIT: First, we must implement compiler rewrite path to unwrap
+// this code.
+AMP.extension('amp-share-tracking', () => {
```

Comment:
It'd be. But I'm not a big fun of that. If someone wants to unit test a specific class - there's no reason why not allow it. All I want to make sure is that we don't just install the whole thing into global window on `import` - that would give us proper hermetic tests.